main.nf

```diff
@@ -551,6 +594,7 @@ process merge_openswath_output {
      """
      pyprophet merge --template=${lib_file_template} \\
                      --out=${Sample}_osw_file_merged.osw \\
+                     --no-same_run \\
```

Comment:
I think it would be beneficial if the process documentation could give hints why e.g, this parameter needs to be set.